<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueAssistantDialog</class>
 <widget class="QDialog" name="IssueAssistantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Issue assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="subHeadlineLabel">
     <property name="text">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This assistant will help you to ask &lt;span style=&quot; font-weight:600;&quot;&gt;questions&lt;/span&gt;, post &lt;span style=&quot; font-weight:600;&quot;&gt;ideas&lt;/span&gt; or report &lt;span style=&quot; font-weight:600;&quot;&gt;problems&lt;/span&gt; on &lt;a href=&quot;https://github.com/pbek/QOwnNotes/issues/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;GitHub&lt;/span&gt;&lt;/a&gt;. Please write plain and &lt;span style=&quot; font-weight:600;&quot;&gt;simple English&lt;/span&gt; sentences so that you can understood by everyone. And please keep the text as &lt;span style=&quot; font-weight:600;&quot;&gt;short as possible&lt;/span&gt;!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">&amp;Back</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/go-previous.svg</normaloff>:/icons/breeze-qownnotes/16x16/go-previous.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string notr="true">&amp;Next</string>
        </property>
        <property name="icon">
         <iconset theme="go-next" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/go-next.svg</normaloff>:/icons/breeze-qownnotes/16x16/go-next.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string notr="true">&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="issuePage">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string notr="true">Title</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="titleLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string notr="true">Type of issue</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="issueTypeComboBox">
            <item>
             <property name="text">
              <string notr="true">Question</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Feature request</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Problem</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="expectedBehaviourGroupBox">
         <property name="title">
          <string notr="true">Expected behaviour</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QOwnNotesMarkdownTextEdit" name="expectedBehaviourPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="questionGroupBox">
         <property name="title">
          <string notr="true">State your question</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QOwnNotesMarkdownTextEdit" name="questionPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QGroupBox" name="stepsGroupBox">
         <property name="title">
          <string notr="true">Steps to reproduce</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QOwnNotesMarkdownTextEdit" name="stepsPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QGroupBox" name="actualBehaviourGroupBox">
         <property name="title">
          <string notr="true">Actual behaviour</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QOwnNotesMarkdownTextEdit" name="actualBehaviourPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logOutputPagePage">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string notr="true">Log output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QPushButton" name="refreshLogButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QOwnNotesMarkdownTextEdit" name="logOutputPlainTextEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string notr="true">Please make sure that the log output provides useful information.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="debugSettingsPagePage">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string notr="true">Information from the Debug settings page</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0" rowspan="3" colspan="2">
           <widget class="QOwnNotesMarkdownTextEdit" name="debugOutputPlainTextEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string notr="true">If you want you can remove</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="submitPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Text to submit</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QOwnNotesMarkdownTextEdit" name="bodyPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="postButton">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Submit issue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Title</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="submitTitleLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="headlineLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Issue assistant</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QOwnNotesMarkdownTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">widgets/qownnotesmarkdowntextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
  <include location="../breeze-qownnotes.qrc"/>
 </resources>
 <connections/>
</ui>
